doc/api-extensions.md

```diff
@@ -2289,3 +2289,7 @@ Check the list of {doc}`configuration options </config-options>` to see which co
 ## `syslog_socket`
 
 This introduces a syslog socket that can receive syslog formatted log messages. These can be viewed in the events API and `lxc monitor`, and can be forwarded to Loki. To enable this feature, set `core.syslog_socket` to `true`.
+
+## `event_lifecycle_name_and_project`
+
+This adds the fields `Name` and `Project` to life-cycle events.
```

Comment:
I preferred the `lifecycle` spelling. Make it easier to grep/search. Presumably you added the hyphen to silence the spellchecker? If that was the case, I'd probably tell it to ignore that one.

Reply:
Yeah, the spell checker was complaining. Also life-cycle is mentioned in this document already so we should keep it consistent.

Reply:
lifecycle: 5 matches, life-cycle: 2 matches :)

Reply:
Oh damn, we aren't consistent :D That's because the API extension names don't include hyphens, and if it's quoted, spellcheck will ignore it.

Reply:
Let me fix that.